xtext/org.icyphy.linguafranca.tests/src/org/icyphy/tests/LinguaFrancaValidationTest.xtend

```diff
@@ -787,14 +799,212 @@ class LinguaFrancaValidationTest {
         ]
     }
     
-    @Test
-    def void checkTargetProperties() {
-        parseWithoutError('''
-                target C {coordination: centralized};
+    /**
+     * Maps a type to a list of known good values.
+     */
+    val primitiveTypeToKnownGood = #{
```

Comment:
@Soroosh129, @lsk567: these lists deserve scrutiny. I'm pretty certain that these don't cover all the cases. If you find any missing, please add them.